test/cql-pytest/test_guardrail_compact_storage.py

```diff
@@ -0,0 +1,32 @@
+import pytest
+from util import config_value_context, new_test_keyspace, new_test_table
+from cassandra_tests.porting import *
+
+# Tests for the enable_create_table_with_compact_storage guardrail.
+# Because this feature does not exist in Cassandra , *all* tests in this file are
+# Scylla-only. Let's mark them all scylla_only with an autouse fixture:
+@pytest.fixture(scope="function", autouse=True)
+def all_tests_are_scylla_only(scylla_only):
+    pass
+
+def test_create_table_with_compact_storage_default_config(cql, this_dc):
+    ks_defs = f"WITH REPLICATION = {{ 'class' : 'NetworkTopologyStrategy', '{this_dc}' : 1 }}"
+    with new_test_keyspace(cql, ks_defs) as test_keyspace:
```

Comment:
You're working too hard to create test_keyspace - you can replace all this `this_dc` and `new_test_keyspace()` business by just the existing "test_keyspace" fixture :-) Much shorter, and faster.
You only need to create a new test_keyspace if you're actually testing CREATE KEYSPACE - which you aren't.

Reply:
Will be fixed in v3